extensions/wikia/CuratedContent/js/CuratedContentManagmentTool.js

```diff
@@ -61,28 +68,46 @@ $(function () {
 
 				$ul.sortable('refresh');
 			},
-			checkInputs = function (elements, checkEmpty, required) {
-				var names = [];
+			/**
+			 * Validate input elements
+			 *
+			 * @param elements
+			 * @param options array consists of ['checkEmpty', 'required', 'limit']
+			 */
+			checkInputs = function (elements, options) {
+				var names = [],
+					optionCheckEmpty, optionRequired, optionLimit;
+
+				optionCheckEmpty = typeof options !== 'undefined' ? (options.indexOf('checkEmpty') !== -1) : false;
+				optionRequired = typeof options !== 'undefined' ? (options.indexOf('required') !== -1) : false;
+				optionLimit = typeof options !== 'undefined' ? (options.indexOf('limit') !== -1) : false;
 
 				elements.each(function () {
 					var val = this.value,
 						$this = $(this);
 
-					if (required && val === '') {
+					if (optionRequired && val === '') {
 						$this
 							.addClass('error')
 							.popover('destroy')
 							.popover({
 								content: requiredError
 							});
-					} else if (!~names.indexOf(val)) {
+					} else if (optionLimit && val.length > 48) {
+						$this
+							.addClass('error')
+							.popover('destroy')
+							.popover({
+								content: tooLongLabelError
+							});
+					} else if (names.indexOf(val) === -1) {
 						names.push(val);
```

Comment:
could you add more descriptive name to names array. It's hard to tell what it does on the first sight.